services/plugins/generic_plugins/parsers.go

```diff
@@ -0,0 +1,180 @@
+package generic_plugins
+
+import (
+	"crypto/sha256"
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"time"
+
```

Comment:
empty line

Reply:
no need as it's done like this in other files